rfi_file_monitor/applicationwindow.py

```diff
@@ -256,6 +256,28 @@ def __init__(self, *args, **kwargs):
                 hexpand=True, vexpand=False,
         ), 'allowed_patterns')
         allowed_patterns_grid.attach(self._allowed_patterns_entry, 1, 0, 1, 1)
+        self._add_advanced_options_horizontal_separator()
```

Comment:
```suggestion
        
        self._add_advanced_options_horizontal_separator()
        
```

Add newlines before and after to improve readability. I will be really happy when we can move the advanced options to its own class and file!